### ISSUE TITLE
channel description right

### ISSUE THREAD

**Issue by preterive** [mumble-voip/mumble#3386]
Hello,
could you please add a "box" at the right side for the channel description?

**Comment by Kissaki** [comment]
I don't understand. Channels already have a box icon to their right that show the channel description when clicking on it, same as user comments. Is that not what you mean?

**Comment by mkrautz** [comment]
He wants a box like TeamSpeak, I think.

**Comment by Krzmbrzl** [comment]
I support this request. It could be a "multi-purpose" detail window that shows channel description if a channel is selected and a user's comment if a user is selected

**Comment by jvbsl** [comment]
I've started a little work on that
![image](https://github.com/mumble-voip/mumble/assets/13928225/c786fb00-09d9-4518-bf90-bc06c7b309da)

I also wanted to show things like client version, connection time/idle time for the users
![image](https://github.com/mumble-voip/mumble/assets/13928225/2031853f-f240-4266-98a3-cc427e12d8be)

But then I noticed that this information is not readily available instead it needs to be polled. Is there a reason why it wasn't implemented that the idle state changed is pushed instead? E.g. including a timestamp idle since when, then the clients would only need that information and if they have the timestamp information they can then calculate for how long the user was online/idle instead of re-requesting the information from the server every time.
Any reason to not implement it that way in addition to the current way(for backwards compatibility)?

**Comment by Krzmbrzl** [comment]
> Is there a reason why it wasn't implemented that the idle state changed is pushed instead?

Well, I guess there simply was a need for this information on the client-side and thus the information is kept on the server :thinking: 

>  Any reason to not implement it that way in addition to the current way(for backwards compatibility)?

In terms of backwards-compatibility, this shouldn't be an issue.

However, I haven't quite understood what you need this information for. What do you want to do with it? :thinking:

**Comment by jvbsl** [comment]
I would like to show this information about users in the above view, as well as perhaps add it in the talking ui, to know how long someone was idle, but then polling would be a lot of unneeded traffic.
Also it would make the user information dialog more responsive If we'd use it there as well.

So if that idle communication is something that would be merged, then I will work on that in a separate PR

**Comment by Krzmbrzl** [comment]
There is also the issue of privacy that needs to be considered. How long someone is idling already could be viewed as information thatbis nobody else's business. 

I'm not convinced that the idle time is an important enough information to go through this hassle.
I would vote for implementing the channel boxes without idle time for now. If we later consider it important enough, we can still add it.

**Comment by RafGamer** [comment]
@jvbsl are you still working on that? I would really like to see this feature implemented.

**Comment by jvbsl** [comment]
Well you can't really call it working on it. The thing is the buildsystem isn't working to well on my local PC because I often have libraries which are "too new", that's why I started to work on refactoring the build system, which in turn is blocked by: https://github.com/microsoft/vcpkg/pull/33648
But I started to do some work on that last week to try to get the current state working with mumble, then I would push these changes to vcpkg.

But I can also try how well the buildsystem in it's current state runs on my PC and check whether I can get this done without changing the PR, (If I do forget to answer, do not have any qualms pinging me^^)
